### PR TITLE
feat: add countdown subpage

### PR DESCRIPTION
**Pull request by jianeasy**
## Owner

- [x] Nickname(Tuya): 艾黎
- [x] Intra-group review（组内审阅人员）：萧瑟

## Description

tuya-panel-lamp-sdk add countdown-subpage

## Package

Please select the relevant package.

- [x] tuya-panel-lamp-sdk

## Type of change

Please select the relevant option.

- [x] New feature (non-breaking change which adds functionality)

## Self CheckList

Please check all options.

- [x] Intra-group review
- [x] ESLint lint
- [x] Example Run
- [x] Jest Run
- [x] No Grammer Error
- [x] No Spell Error
- [x] No Merge Conflict

### CODE REVIEW

File: example/src/scenes/Lamp/CountdownSubPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Utils } from 'tuya-panel-kit';
+import Strings from '../../../i18n';
+import CountdownSubPage from '../../../../../packages/tuya-panel-lamp-sdk/src/components/time/countdown-subpage';
```
Comment (XiaooSe) [inline example/src/scenes/Lamp/CountdownSubPage/index.tsx:4]: 用包名引用

File: packages/tuya-panel-lamp-sdk/src/components/time/countdown-subpage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    alignItems: 'center',
+    flex: 1,
+    justifyContent: 'center',
+    width: '100%',
+  },
+  subTitle: {
+    fontSize: cx(10),
+    opacity: 0.5,
+    textAlign: 'center',
+    width: winWidth - cx(68) * 2,
```
Comment (XiaooSe) [inline packages/tuya-panel-lamp-sdk/src/components/time/countdown-subpage/index.tsx:176]: 布局使用padding 隔离空间，不应硬算

File: packages/tuya-panel-lamp-sdk/src/components/time/countdown-subpage/interface.ts [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+   */
+  cancelButtonStyle: StyleProp<ViewStyle>;
+  /**
+   * 取消文本
+   */
+  cancelText: string;
+  /**
+   * 取消文本样式
+   */
+  cancelTextStyle: StyleProp<TextStyle>; // 取消文本样式
```
Comment (XiaooSe) [inline packages/tuya-panel-lamp-sdk/src/components/time/countdown-subpage/interface.ts:67]: 文本注释多余

**Comment by XiaooSe** [comment]
LGTM

File: packages/tuya-panel-lamp-sdk/src/components/time/countdown-subpage/interface.ts [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+   */
+  isShowHour: boolean;
+  /**
+   * 子标题文本样式
+   */
+  subTitleStyle: StyleProp<TextStyle>;
+  /**
+   * 时间文本样式
+   */
+  timeTextStyle: any;
```
Comment (paruki-gin) [inline packages/tuya-panel-lamp-sdk/src/components/time/countdown-subpage/interface.ts:159]: timeTextStyle requires a type definition

File: packages/tuya-panel-lamp-sdk/src/components/time/countdown-subpage/timePicker/index.tsx [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+      setSecond(second_);
+    }
+  }, [time]);
+
+  const handleChange = () => {
+    onChange(Number(hour) * 3600 + Number(minute) * 60 + Number(second));
+  };
+
+  const handleHour = (value: number) => {
+    const state: any = { hour: formatValue(value) };
```
Comment (paruki-gin) [inline packages/tuya-panel-lamp-sdk/src/components/time/countdown-subpage/timePicker/index.tsx:53]: state requires a type definition

**Comment by youngjuning** [comment]
LGTM~

**Pull request merged**